eradicate.py

```diff
@@ -28,176 +28,204 @@
 import io
 import os
 import re
-import sys
 import tokenize
 
 __version__ = '1.0'
 
+class Eradicator(object):
+    """Eradicate comments."""
+    MULTILINE_ASSIGNMENT_REGEX = re.compile(r'^\s*\w+\s*=.*[(\[{]$')
+    PARTIAL_DICTIONARY_REGEX = re.compile(r'^\s*[\'"]\w+[\'"]\s*:.+[,{]\s*$')
+    CODING_COMMENT_REGEX = re.compile(r'.*?coding[:=][ \t]*([-_.a-zA-Z0-9]+)')
+
+    DEF_STATEMENT_REGEX = re.compile(r"def .+\)[\s]+->[\s]+[a-zA-Z_][a-zA-Z0-9_]*:$")
+    WITH_STATEMENT_REGEX = re.compile(r"with .+ as [a-zA-Z_][a-zA-Z0-9_]*:$")
+    FOR_STATEMENT_REGEX = re.compile(r"for [a-zA-Z_][a-zA-Z0-9_]* in .+:$")
+
+    CODE_INDICATORS = list('()[]{}:=%') + ['print', 'return', 'break', 'continue',
+                                           'import']
+    CODE_KEYWORDS = [r'elif\s+.*', 'else', 'try', 'finally', r'except\s+.*']
+    CODE_KEYWORDS_AGGR = CODE_KEYWORDS + [r'if\s+.*']
+
+    DEFAULT_WHITELIST = (
+        r'pylint',
+        r'pyright',
+        r'noqa',
+        r'type:\s*ignore',
+        r'fmt:\s*(on|off)',
+        r'TODO',
+        r'FIXME',
+        r'XXX'
+    )
+    WHITELIST_REGEX = re.compile(r'|'.join(DEFAULT_WHITELIST), flags=re.IGNORECASE)
```

Comment:
Let's try to clean all unused `r` prefixes.

Reply:
For me the `r` prefix triggers regex syntax highlighting .. else it is only a string. Even for the example you picked here the coloring of the pipe changes.
So I would like to keep them for the regexes. But if you still want them removed I can do.

Reply:
Ok 👍